src/shared/ui/Modal/index.tsx

```diff
@@ -0,0 +1,64 @@
+import styles from './style.module.scss';
+import { Fragment, ReactNode } from 'react';
+import { ModalProps, HeaderProps, Children, ButtonProps } from './index.type';
+
+const Modal = ({ children, isOpen }: ModalProps): JSX.Element => {
+  return (
+    <>
+      {isOpen ? (
+        <div className={styles.background}>
+          <LayOut>{children}</LayOut>
+        </div>
+      ) : null}
+    </>
+  );
+};
+
+const LayOut = ({ children }: Children) => {
+  return <div className={styles.layout}>{children}</div>;
+};
+
+const Title = ({ children }: { children: string }) => {
+  return <h1>{children}</h1>;
+};
+
+const Description = ({ children }: { children: string }) => {
+  return <p>{children}</p>;
+};
+
+const Header = ({
+  Title,
+  Description,
+  btnChildren,
+  handleClick,
+}: HeaderProps) => {
+  return (
+    <div>
+      {Title}
+      {Description}
```

Comment:
```suggestion
      <Title />
      <Description />
```

❓ 
> 짧은 정보
> `{Title}` 은 함수이고
> `{Title()}` === `<Title />` 호출문은 jsx 문과 동일합니다. 

그런데 JSX에서 함수만 두고 호출을 하지 않으면(현재상태) UI에 렌더링이 안될 것 같은데 잘 되던가요..?

Reply:
안그래도 빌딩만 되고 서버 돌아가는걸 못 보고 올렸을 때라 확인을 못했네요! 이번에 수정 후 돌아가게 올리겠습니다!